enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java

```diff
@@ -721,6 +721,9 @@ public NearbyLocationsResult queryNearbyLocations(final EnumSet<LocationType> ty
 
         try {
             final JSONObject head = new JSONObject(page.toString());
+            if (head.has("error")) {
+                return new NearbyLocationsResult(resultHeader, Status.INVALID_ID);
+            }
```

Comment:
One of the live tests with an invalid station arrived here and `head` had no `pagination`, but an `error`.

Reply:
Maybe this change should go into a different PR that implements error handling for all Navitia calls?